src/RichText/RichText.php

```diff
@@ -535,7 +535,7 @@ private static function getHtmlSanitizer(): HtmlSanitizer
             ])
             ->allowRelativeLinks()
             ->allowRelativeMedias()
-            ->withMaxInputLength(pow(2, 24)) // 2^24 corresponds to MySQL MEDIUMTEXT max length
+            ->withMaxInputLength(-1)
```

Comment:
As it is now possible to use an unlimited input length, I removed the previous limit that was arbitrary. It is not the responsability of this class to determine whether the input/result is too length.